Qubes OS Summit: History from organizer's perspective (#100)
QubesOS/qubes-posts: 2022-08-07-qubes-os-summit-history.md

---
layout: post
title: "Qubes OS Summit: History from organizer's perspective"

> categories: talks

Reviewer: Please add an "author" field, like this:

```
author: <name>
```

`<name>` can be `3mdeb`, a comma-separated list of your personal names, or whatever else is appropriate.

Author: ```suggestion
categories: talks
author: Michał Żygowski
```

Author: Added

---

> The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held in Berlin from September 9th to 11th in hybrid format, in person and live-streamed for remote access. More details [here](https://qubesos.3mdeb.com/). Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.

Reviewer: Before this is published, I will be adding a short editor's note at the very top that incorporates some of this information so that readers understand that this is a guest post ([example](https://www.qubes-os.org/news/2022/04/10/windows-integration-by-tabit-pro/), so there is no need to self-preface. You can instead simply start at the beginning of your topic, if you prefer.

Author: Moved to a separate section.

didn't stop us from holding the event again! It even gave us a better
opportunity to attract and gather more people virtually to attend the event.
Thanks to our 3mdeb colleagues, who set up the live recording and streaming of
the whole Qubes OS 2020 Summit, it is possible to see and listen to the talks
given by Qubes OS and 3mdeb experts on
[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIwhPXcuYKtS50CHQOvt_BO)
over and over again. From 3mdeb side we continued firmware-related topics, fpr
example first time Qubes OS Anti Evil Maid on AMD platform. It was quite a
challenge for us because we have organized such a virtual event for the first
time. Fortunately, everything went smoothly and finished with success after 4
sessions 2.5 hours long.

In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
team, still virtually, unfortunately. However, this year brought more speakers
from other companies and projects, which we were very happy about. There were 2
sessions almost 4 hours each and one may watch the talks at
[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).

> This time the 3mdeb's stage belonged to Piotr Król (CEO of 3mdeb) who presented

Reviewer 2: ```suggestion
This time, the 3mdeb's stage belonged to Piotr Król (CEO of 3mdeb) who presented
```

his adventures using Qubes OS as an everyday device, talking about USB camera
and cryptocurrency wallets. Of course firmware-related topics could not be
missing and Piotr also showed continuation of his previous year's efforts of
securing the VMs with SRTM and Secure Boot.

> # The future of Qubes OSe Summits

Reviewer: Typo: `OSe`

topics on the verge of firmware and operating systems. All this time we have
been gradually building plans and ideas, and getting closer and closer to the
goal of improving the security of Qubes OS, especially in areas we feel best
at, i.e. SRTM and DRTM (AEM). In the upcoming event, we have prepared a few
interesting projects and ideas around Anti Evil Maid support and open-source
firmware on modern hardware.
[Here](https://cfp.3mdeb.com/qubes-os-summit-2022/schedule/#) you may find the
full schedule. Among others we welcome the following speakers:

* Wessel Klein Snakenborg from Novacustom with a story of open-source on modern
  laptops as a candidate to Qubes OS certification
* Brent Cowing from Protectli talking about security of his small form factor
  computers also running open-source firmware
* Jan Suhr from Nitrokey talking about how enterprise requirements tied to
  Windows can be met with Qubes OS
* Arthur Rasmusson from Arc Compute talking about GPU virtualization and its
  possible benefits to Qubes oS
* Puck Meerburg from Spectrum OS telling about the power of Wayland for GUI
  isolation and how it benefits systems like Qubes OS or Spectrum OS

> * And and everyone from 3mdeb, Qubes OS team and others

Reviewer 2: And and

Author: ```suggestion
* And everyone from 3mdeb, Qubes OS team, and others
```

---
layout: post
title: "Qubes OS Summit: History from organizer's perspective"

> categories: talks

Author: @mfc I'm wondering if I chose the category correctly. Wouldn't the article category be more appropriate?

Reviewer: You can use both:

```
categories:
- talks
- articles
```

Author: ```suggestion
categories:
- talks
- articles
```

---

> The next [Qubes OS Summit 2022](https://www.qubes-os.org/news/2022/07/29/qubes-os-summit-2022/) edition is upcoming. This year it will be held live in Berlin from September 9th to 11th. Don't miss the event and more importantly how it started. In the article [Michał Żygowski from 3mdeb](https://blog.3mdeb.com/authors/michal-zygowski/) (Qubes OS Summit co-organizer) will describe the history and organizer's perspective of the event.

Reviewer 3: probably worth mentioning here that it will be live-streamed and accessible remotely

Reviewer 3: and linking to the actual summit website with more info

Author: Done

# How did all of this start?

In May 2019 3mdeb funded Linux training for its employees, having no idea who
the instructor will be. At some point, it occurred that the mysterious
instructor will be none other than Marek Marczykowski-Górecki, the same person
who leads the Qubes OS project. As huge fans of Qubes OS and its approach to
security, as also fans of Joanna Rutkowska security research in the area of
firmware and computer architecture, we were delighted to meet him in person in
our office in Gdańsk, Poland. The time has come when the idea of a short event
focused on firmware impact on Qubes OS security has been born. We asked Marek
if he has some spare time to discuss firmware impacts on Qubes OS security and
he liked the idea. The event has been called humorously a "minisummit". That is
how the first Qubes OS summit started, one day, in a small conference room,
with a tiny group of people passionate about firmware and system security,
nothing outrageous (in the meaning of scale). One may still read what topics
have been discussed

> [here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/).

Reviewer 4: @miczyg1 very well written IMHO.  Please add topic examples shortly here and in other places, so one without visit could realize what we talking about during such events.

Author: Done for all previosu editions.

Another May has come, the year 2020, the pandemic dominated the world. But it
didn't stop us from holding the event again! It even gave us a better
opportunity to attract and gather more people virtually to attend the event.
Thanks to our 3mdeb colleagues, who set up the live recording and streaming of
the whole Qubes OS 2020 Summit, it is possible to see and listen to the talks
given by Qubes OS and 3mdeb experts on
[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIwhPXcuYKtS50CHQOvt_BO)
over and over again. It was quite a challenge for us because we have organized
such a virtual event for the first time. Fortunately, everything went smoothly
and finished with success after 4 sessions 2.5 hours long.

In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
team, still virtually, unfortunately. However, this year brought more speakers
from other companies and projects, which we were very happy about. There were 2
sessions almost 4 hours each and one may watch the talks at
[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).

# The future of Qubes OSe Summits

> Finally, the time has come when we may meet together in person, after two

Reviewer 4: two? not three? in person event was in 2019, we saw Marek during FOSDEM 2020 mabe, so at least `over two years`. When I'm counting my personal attendence last time it was Embedded World in February 2020.

Author: Yes, three. Fixed.

years! The Qubes OS Summit 2022 is fast approaching, with only a few days left
before the start of the event in Berlin. As it is no longer held only virtually

> (but still possible to attend remotely for those who can be with us in person)

Reviewer 3: can -> can't

Author: Fixed

the organization is more complicated, both from the logistics and presentations
side. Every year we try our best to dig into hardcore security topics on the
verge of firmware and operating systems. All this time we have been gradually
building plans and ideas, and getting closer and closer to the goal of
improving the security of Qubes OS, especially in areas we feel best at, i.e.
SRTM and DRTM (AEM). In the upcoming event, we have prepared a few interesting
projects and ideas around Anti Evil Maid support and open-source firmware on
modern hardware. [Here](https://cfp.3mdeb.com/qubes-os-summit-2022/schedule/#)
you may find the full schedule. Among others we welcome the following speakers:

* Wessel Klein Snakenborg from Novacustom with a story of open-source on modern
  laptops as a candidate to Qubes OS certification
* Brent Cowing from Protectli talking about security of his small form factor
  computers also running open-source firmware
* Jan Suhr from Nitrokey talking about how enterprise requirements tied to
  Windows can be met with Qubes OS

> * Arthur Rasmusson from OpenMdev project about GPU virtualization and its

Reviewer 4: We probably should say Arc Compute here

Author: Fixed

>   possible benefits to QUbes oS

Reviewer 4: ```suggestion
  possible benefits to Qubes OS
```

Author: Fixed

* Puck Meerburg from Spectrum OS telling about the power of Wayland for GUI
  isolation and how it benefits systems like Qubes OS or Spectrum OS
* And and everyone from 3mdeb, Qubes OS team and others

We hope that the event and community around it and the Qubes OS project will

> grow in the future, bringing benefits and collaboration for everybody.

Reviewer 4: ```suggestion
grow in the future, bringing reasonable trustworthiness and collaboration for everybody.
```

?

Author: Applied

Definitely, 3mdeb will try its best to co-organize the event in the upcoming
years.

# Summary

By a mere coincidence, meeting Marek Marczykowski-Górecki gave birth to a
fascinating series of Qubes OS Summits and led us to this day.

> We (3mdeb) are very proud to be the co-organizers of the 4th edition of the

Reviewer 4: Maybe rephrase that in way as follows:

Together with InvisibleThingsLab we (3mdeb) are very proud ....

Author: Done
